File: CS50 2022/lecture2/scores.c
#include <stdio.h>
#include <cs50.h>

const int N = 3;

float average(int array[]);

int main(void)
{
    int scores[N];

    for (int i = 0; i < N; i++)
    {
        scores[i] = get_int("Score: ");
    }
    
    printf("Average: %f\n", average(scores));
}

float average(int array[])
{
    int total = 0;

    for (int i = 0; i < N; i++)
    {
        total += array[i];
    }

    return total / (float) N;
}

File: CS50 2022/lecture6/scores.py
from cs50 import get_int

scores = []
n = get_int('N° of scores: ')
for i in range(n):
    scores.append(get_int('Score: '))
    # scores += [get_int('Score: ')]

avg = sum(scores) / len(scores)
print(f'Average: {avg}')